pkg/termcorder/termcorder_test.go

```diff
@@ -0,0 +1,116 @@
+package termcorder_test
+
+import (
+	"bytes"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/haguro/termcord/pkg/termcorder"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewTermcording(t *testing.T) {
+	t.Parallel()
+	cfg := &termcorder.Config{}
+	tc, err := termcorder.NewTermcording(cfg)
+	assert.IsType(t, &termcorder.Termcording{}, tc)
```

Comment:
It's usually clearer to write something like:

```go
var tc *termcorder.Termcording
```

so that the compiler, not the test machinery, can validate that it's the right type.